lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -401,18 +411,21 @@ def parse(body)
       end
 
       def url(action, authorization)
+        return ogone_direct + uri(action, authorization) if @options[:url_override].to_s == 'ogone_direct'
```

Comment:
`@options[:url_override]` is always populated? 

If the answer is no, I think it will fail when this is `nil`